BlazarUI/app/scripts/actions/buildActions.js

```diff
@@ -34,7 +34,7 @@ function getModule() {
   const modulesPromise = branchModules.fetch();
 
   modulesPromise.done( () => {
-    gitInfo.moduleId = _.find(branchModules.data, (m) => {
+    gitInfo.moduleId = find(branchModules.data, (m) => {
```

Comment:
Do we want to target IE?

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/find